docs/source/running-jobs/environment-variables.rst

```diff
@@ -142,9 +154,9 @@ Environment variables for ``run``
 
        If a task is run as a :ref:`managed spot job <spot-jobs>`, then all
        recoveries of that job will have the same ID value. Read more :ref:`here <spot-jobs-end-to-end>`.
-     - sky-2023-07-06-21-18-31-563597_myclus_id-1
+     - sky-2023-07-06-21-18-31-563597_myclus_1
```

Comment:
How about also including a logical schema for both cases in the cell above? `sky(-managed)-<timestamp>-<cluster_name>_<job_id>(-task_idx)` (please check correctness). This can help users parse it.

Does job_id start with 1 and task_idx 0? May need to update example values.